phpBB/phpbb/db/migration/data/v33x/update_user_lastvisit_data.php

```diff
@@ -0,0 +1,97 @@
+<?php
```

Comment:
Is this migration really needed? Seems like this will be run the first time session_gc will be run anyway?